crowdin.yml

```diff
@@ -1,29 +1,28 @@
 #
 # Your Crowdin credentials
 #
-"project_id": "420172"                    # Crowdin project Settings > API & Webhooks > API > Project Id
-"api_token_env": "CROWDIN_PERSONAL_TOKEN" # Low priority: superseded by user-specific credentials file ~/.crowdin.yaml
-"base_path": "."
-"base_url": "https://api.crowdin.com"
+'project_id': '415116' # Crowdin project Settings > API & Webhooks > API > Project Id
+'api_token_env': 'CROWDIN_API_TOKEN' # Low priority: superseded by user-specific credentials file ~/.crowdin.yaml
+'base_path': '.'
+'base_url': 'https://api.crowdin.com'
 
 #
 # Choose file structure in Crowdin
 # e.g. true or false
 #
-"preserve_hierarchy": true
+'preserve_hierarchy': true
 
 #
 # Files configuration
 #
-files:
-  [
+files: [
     {
-      "source": "/src/locale/en-US.json",
-      "translation": "/src/locale/%locale%.json",
+      'source': '/src/locale/en-US.json',
+      'translation': '/src/locale/%locale%.json',
     },
     # Duplicate file
     {
-      "source": "/src/locale/en.json",
-      "translation": "/src/locale/%two_letters_code%.json",
+      'source': '/src/locale/en.json',
+      'translation': '/src/locale/%two_letters_code%.json',
```

Comment:
Question: what is the purpose of %two_letters_code%?

Reply:
two letter codes as I understand they will take 2 letters short for the language.
I.E: vn
or %locale% to be en-US

Reply:
@sbenmoussati 
for more information
![image](https://user-images.githubusercontent.com/97150869/195299966-1ecb32ec-a234-4283-a00d-7da7084e5b47.png)